config/routes/data.rb

```diff
@@ -173,9 +173,6 @@
 
 resources :contents do
   concerns [:data_routes]
-  collection do
-    get :filter
```

Comment:
How is this being replaced? Or is just not used anymore?

Reply:
It's just `/index.json` (we need to remove all /filter endpoints for this pattern).